.gitignore

```diff
@@ -131,3 +131,4 @@ dmypy.json
 TODO.md
 coverage
 artifacts
+/credentials/
```

Comment:
Can you add the missing trailing newline here?

Reply:
Sure